### ISSUE TITLE
Discontinuity on API and Websockets in regards to Music

### ISSUE THREAD

**Issue by dyslexicjedi** [MediaBrowser/Emby#3233]
Situation: I play a song and stop the song. I make a request to the api for Sessions, it lists the song as a NowPlayingItem with Id "fa"{truncated} (even though it's not playing). Now I play an episode of something, the NowPlayingItem is the episode with the same Id "fa"{truncated}, the websocket notification is fired. I stop the episode, the websocket stop is fired and the API goes back to the NowPlayingItem with Id "fa"{truncated} with no websocket fire. 

Basically it seems like the API is stuck saying the song is playing (it's not) whenever another other item isn't playing.

**Comment by dyslexicjedi** [comment]
Just to follow up, the Dashboard in emby says it's "playing" as well, but it's not.

**Comment by LukePulverenti** [comment]
what app is playing?

**Comment by dyslexicjedi** [comment]
Chrome through the Emby Web Interface. The API says it's in a paused state, my concern is how it reverts back to the song after another object is ended without any other notification. and it doesn't show the song as playing or paused while the video is playing, only after the video is done, does the song show back as playing.

**Comment by LukePulverenti** [comment]
What version of Chrome?

**Comment by dyslexicjedi** [comment]
Version 64.0.3282.140 (Official Build) Built on Ubuntu , running on LinuxMint 18.3 (64-bit)

**Comment by dyslexicjedi** [comment]
I think for now, I'll just program around it. Just filter anything that is MediaType == "Audio" and PlayState.IsPaused == true but just wanted to let you know.

**Comment by LukePulverenti** [comment]
Did you have multiple browser tabs open?

**Comment by dyslexicjedi** [comment]
Multiple tabs open yes, to emby no. Just one tab to the emby web interface.

**Comment by dyslexicjedi** [comment]
I'm able to reproduce it now. So basically, play a song. Pause it. Open a video, play the video, stop the video. The audio controller is gone, but the song is still in a paused state in the background.
